app/views/diabetes-content/diabetes-2-diagnosis.html

```diff
@@ -42,21 +42,19 @@
 Then make another [GP appointment](https://nhsalpha.herokuapp.com/book-an-appointment/start) and take your list of questions with you.
 
 ###Dealing with confusion and shock after the diagnosis
-There’s also a lot of [information on diabetes](/XXX) available where you can read up on the condition.
+There’s also a lot of [information on diabetes](https://nhsalpha.herokuapp.com/diabetes-content/information-and-support-type-2-diabetes) available on:
```

Comment:
Ah okay I see... Maybe "including" then?

Or "...a lot of information on diabetes available including:"

Sorry, doesn't seem to read right at the moment?